src/makeRobotAccountant.js

```diff
@@ -7,6 +7,29 @@
 
 function makeRobotAccountant() {
   // write code here
+  let sum = 0;
+  let counter = 0;
+
+  return (a = 0) => {
+    sum += a;
+
+    return (b = 0) => {
+      sum += b;
+      counter += 1;
+
+      if (counter > 3 && (counter % 2) === 0) {
+        sum = 0;
+
+        return 'Bzzz... Error!';
+      } else {
+        const res = sum;
+
+        sum = 0;
```

Comment:
You set it to 0 in both cases, do you really need this variable?
You can just return a + b, as you have only two operands